mautic/mautic
# Fixed entity sync to catch all records and not just 25 per run

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

If there are more than 25 contacts to update or create, only 25 would be processed per script execution. So it became easy to get backlogged and never catch up. This PR addresses this. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create more than 25 contacts (import?)
2. Setup Salesforce
3. Run `php app/console m:i:s -i Salesforce`
4. Run it a second time. It should create the remainder contacts

#### Steps to test this PR:
1. Repeat the above with new contacts
2. Run the script
3. It should now create all the contacts in a single process
